mentat/session.py

```diff
@@ -45,7 +45,7 @@ def __init__(
 
         cost_tracker = CostTracker()
 
-        code_context = CodeContext(stream, git_root, diff, pr_diff)
+        code_context = CodeContext(stream, git_root, diff, pr_diff, ignore_paths)
```

Comment:
Ignore paths is solely for auto context; exclude paths is for the user included context (although soon these 2 things will be merged anyways)